park_info.clj

```diff
@@ -0,0 +1,367 @@
+(
+    def Bertramka{
+        ;---Basic Info---
+        :food nil
+        :food_extra nil
+        :toilet true
+        :toilet_extra nil
+        :POIs "Cultural monument, classical music concerts, social events, W.A. Mozart Museum" ;POI refers to Point of Interest
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground false
+        :playground_extra: nil
+        ;---Directions---
+        :public_transport "Trams #4, #7, #9, #10, #58, #59"
+        :GPS_coords "50°4’15.00’’ N, 14°23’42.00’’ E"
```

Comment:
Non-standard inverted commas used ('' as opposed to ″).

Reply:
Common problem, pasted straight from website. Does not affect functionality to my knowledge but will fix.